source/partials/drupal/dependencies-compatible.md

```diff
@@ -0,0 +1,29 @@
+---
+contenttype: [partial]
+categories: [--]
+newcms: [drupal, drupal8, drupal9]
+product: [--]
+integration: [--]
+tags: [--]
+reviewed: "2022-11-04"
+---
+
+You must confirm that your site is compatible with the latest version of Drupal before you change the Drupal core version.
+ 
+Review the steps below to confirm compatibility for your site's components.
+
+1. Optional. [Convert your site to Composer](/guides/composer-convert) if you plan to use the [Upgrade Status module](https://www.drupal.org/project/upgrade_status/) to assist in your upgrade.
+
+1. Verify that your hosting environment meets Drupal platform requirements.
+
+1. Update to Drupal 8.x-3.15 or higher.
```

Comment:
That's a module version number, which isn't appropriate here. I'd say "Upgrade to the most recent minor release of the version of Drupal that you're currently on."

Reply:
Upgrade to latest is a step further down; you don't need both.